classes/Tools.php

```diff
@@ -1457,15 +1457,7 @@ public static function truncateString($text, $length = 120, $options = array())
 
 	public static function normalizeDirectory($directory)
 	{
-		$last = $directory[strlen($directory) - 1];
-
-		if (in_array($last, array('/', '\\')))
-		{
-			$directory[strlen($directory) - 1] = DIRECTORY_SEPARATOR;
-			return $directory;
-		}
-
-		$directory .= DIRECTORY_SEPARATOR;
+		$directory = $directory = rtrim($directory, '/\\').DIRECTORY_SEPARATOR;;
```

Comment:
Hi,

Thank you for your PR, can you also remove double variable assignation ?

Best regards.